ci/release/update-version.sh

```diff
@@ -75,8 +75,26 @@ sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/cudf/source/
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/dask_cudf/source/conf.py
 sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/dask_cudf/source/conf.py
 
+DEPENDENCIES=(
+  cudf
+  cudf_kafka
+  custreamz
+  dask-cuda
+  dask-cudf
+  kvikio
+  libkvikio
+  librmm
+  rmm
+)
+for DEP in "${DEPENDENCIES[@]}"; do
+  for FILE in dependencies.yaml conda/environments/*.yaml; do
+    sed_runner "/-.* ${DEP}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}\.*/g" ${FILE}
+  done
+  for FILE in python/*/pyproject.toml; do
+    sed_runner "s/${DEP}==.*\",/${DEP}==${NEXT_SHORT_TAG_PEP440}.*\",/g" ${FILE}
+  done
+done
 
-# bump rmm & dask-cuda
 for FILE in conda/environments/*.yaml dependencies.yaml; do
```

Comment:
I totally goofed here. I made these changes twice because I accidentally ran `git restore .` while testing the version changes... and tossed my changes to `update-version.sh` too. I'll review this more carefully and make sure the now-redundant parts are removed.

Reply:
Should be fixed in b07b3fb. Thanks AJ!